tests/tests.sh

```diff
@@ -158,15 +165,15 @@ function compare_to_result()
 	local sav="$2"
 	if [ ! -r "$sav" ]; then return 0; fi
 	test "`cat -- "$sav"`" = "UNDEFINED" && return 1
-	diff -u -- "$sav" "$outdir/$pattern.res" > "$outdir/$pattern.diff.sav.log"
+	diff -u --strip-trailing-cr -- "$sav" "$outdir/$pattern.res" > "$outdir/$pattern.diff.sav.log"
```

Comment:
Is there a test in particular where the trailing CR is significant?

If not, at this point I'd rather get the ~1000 tests working on Windows than preemptively support a future test. We can always update the script if it becomes an issue or when we have an official policy/services for line endings.